### PR TITLE
API endpoint and test for listing all inventory records

### PR DESCRIPTION
**Pull request by aayushagrawal135**
- Route for listing all inventory records. `GET /inventory-records`.
- Test for
    - Asserting count of listed records.
    - Asserting values of listed records (irrespective of order)

### CODE REVIEW

**Review by sakshambassi — APPROVED** [review]
lgtm! ship it

**Review by rishit13 — APPROVED** [review]
Looks good !

**Review by rishit13 — APPROVED** [review]
Looks good !

File: tests/test_routes.py [hunk 1]
```diff
@@ -81,18 +90,6 @@ def test_create_inventory_records(self):
         self.assertEqual(new_record["reorder_quantity"], test_record.reorder_quantity)
         self.assertEqual(new_record["restock_level"], test_record.restock_level)
 
-        #uncomment this once list all products works
```
Comment (richa15997) [inline tests/test_routes.py:84]: actually this was used to check the location header is correct, which could have only been done if we could return the record with location properties. we can leave this commented until we have a function that gets data from the database using a filter

Comment (aayushagrawal135) [reply tests/test_routes.py]: Reverted the change!

Comment (richa15997) [reply tests/test_routes.py]: Thanks!

**Review by richa15997 — APPROVED** [review]
Good work!

**Review by rishit13 — APPROVED** [review]

**Pull request merged**